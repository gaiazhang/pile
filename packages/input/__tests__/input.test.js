import React from 'react';
import { shallow } from 'enzyme';
import Input from '../src';

describe('Input suite', () => {
  it('render a input', () => {
    const wrapper = shallow(<Input />);
    expect(wrapper).toMatchSnapshot();
  });
});
