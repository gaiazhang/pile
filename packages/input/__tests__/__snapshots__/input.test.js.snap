// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input suite render a input 1`] = `
<div
  className="pile-input-item"
>
  <InputLabel
    labeltext=""
    prefixCls="pile"
  />
  <InputContent
    autoFocus={false}
    clearable={true}
    disabled={false}
    name=""
    placeholder=""
    prefixCls="pile"
    type="text"
  />
</div>
`;
